source/dotnet/AdaptiveCards.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27004.2005
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29025.244
```

Comment:
we're not ready to support VS2019 at this time :(

Reply:
According to this page we should use .net core 3.0 with the latest preview of Visual Studio, I could try Visual Studio 17 to check if it works too there

https://dotnet.microsoft.com/download/dotnet-core/3.0

Reply:
But according to this one we must use 2019

> Prerequisites for .NET Core 3.0 Preview 3
> 
> .NET Core 3.0 Preview 3 has the same prerequisites as other versions of .NET Core. However, if you want to use Visual Studio to create .NET Core 3.0 projects, you must use the Visual Studio 2019. Visual Studio 2019 can be installed side-by-side with other versions of Visual Studio without conflict.

https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites?tabs=netcore2x

Reply:
@paulcam206, @almedina-ms what happened around this one? Are we ready to roll up to VS 2019? Let's try to close on this one!